client/src/views/Models/Model.vue

```diff
@@ -396,20 +195,59 @@
     drilldownRelatedParameters: any = null;
     drilldownParameters: any = null;
 
-    isSplitView = false;
     subgraph: GraphInterface = null;
+    querygraph: any = null;
```

Comment:
`querygraph` should be of type `SubgraphInterface`; further `subgraph`, `querygraph` and `highlights` seem to be related. Can these be merged?

Reply:
Let's do this while handling merge conflicts with main as I believe some of these are being modified.